content/docs/user-guide/components/reference/physx/prismatic-joint.md

```diff
@@ -0,0 +1,65 @@
+---
+linkTitle: PhysX Prismatic Joint
+description: ' Lean to use the Open 3D Engine PhysX Prismatic Joint component. '
+title: PhysX Prismatic Joint Component
+---
+
+
+
+With the **PhysX Prismatic Joint** component, you can create a dynamic prismatic joint that constrains an entity to the joint, keeping the same rotation but allowing it to move freely along one axis.
+
+## PhysX Prismatic Joint component properties 
+
+![Properties of the PhysX Prismatic Joint component](/images/user-guide/physx/physx/ui-physx-prismatic-joint-component.png)
+
```

Comment:
Note on style. Many of the components in this section have not been updated to the latest style (component property lists in tables such as the [Actor](https://www.o3de.org/docs/user-guide/components/reference/animation/actor/) component). We can merge this topic as is and update the format later since time is an issue.